portal/src/texts/core/elevation.md

```diff
@@ -1,3 +1,17 @@
 ## Elevasjon
 
-Vi ønsker at brukerne lett skal se hvilket innhold som er på ett nivå og hvilket som er over- eller underordnet. Til det bruker vi blant annet skygger, som skaper inntrykk av elevasjon. Vi bruker ulike skygger for å skape et hierarki.
+Prinsippet elevasjon handler om å løfte det som er viktig og skape et intuitivt informasjonshierarki. Det kan blant annet gjøres ved å jobbe med nivåer og dybde.
+
+Bruk av skygge indikerer at elementer blir aktivert. Forskyvning ved scroll skaper også dybde og legger viktig innhold høyere opp og nærmere brukeren. Man kan også skape nivåer ved bruk av tydelige størrelsesforhold mellom elementer.
```

Comment:
Forslag: Vi bruker skygge til å indikere at et element blir aktivt. Vi skaper også dybde ved at et element forskyver seg ved rulling og flytter innholdet høyere opp og nærmere brukeren. Vi kan også skape nivåer ved å lage tydelige størrelsesforhold mellom elementer.

Reply:
Jeg er litt uenig i denne. For det første synes jeg ikke _rulling_ er en god erstatning for _scroll_. Jeg ser da for meg at elementene spinner og ruller over skjermen. NAOB foreslår [_skrolle_](https://www.naob.no/ordbok/skrolle_3), og SNL bruker [det engelske ordet](https://www.naob.no/ordbok/skrolle_3). I tillegg blir det litt feil at vi _også_ flytter innholdet høyere opp, da tanken er at det er forskyvningen som skaper illusjonen av at det er nærmere 😊

Reply:
Scroll og scroll bar har godt innarbeidede oversettelser hos Microsoft, som er de som har den mest omfattende GUI-terminologilisten på norsk. For oss som har skrevet og oversatt dokumentasjon til programvare lenge, så er ikke dette omdiskutert - det heter rulle og rullefelt, og scroll box heter f eks rulelboks. Der må bestemme hva dere vil bruke, men min anbefaling er å følge GUI-terminologien som er utarbeidet på norsk. Rulle for scroll har vært brukt helt siden Windows kom med rullefunksjon.

For det andre du nevnte: Kan det gå an med "...ved at et element forskyver seg og skaper et inntrykk av at det kommer nærmere brukeren"?

Reply:
Skrevet om nå:

> Vi skaper dybde ved hjelp av parallakse: Viktig innhold beveger seg raskere enn bakgrunnen for å skape inntrykk av å ligge nærme brukeren.

Første og siste setning er som du foreslo.

Reply:
Nesten helt enig :-): Vi skaper dybde ved hjelp av parallakse: Viktig innhold beveger seg raskere enn bakgrunnen for å skape inntrykk av at det ligger nærmere brukeren.